services/QuillLMS/spec/controllers/teacher_fix_controller_spec.rb

```diff
@@ -161,6 +161,11 @@
           post :recover_activity_sessions, params: { email: user.email, unit_name: "some name" }
           expect(classroom_unit.reload.assigned_student_ids).to include(another_user.id, other_student.id)
         end
+
+        it 'should touch ActivitySessions to bubble up touch for cache invalidation purposes' do
+          expect_any_instance_of(ClassroomUnit).to receive(:touch)
```

Comment:
Should this be `ActivitySession` instead of `ClassroomUnit`?

Reply:
This is intentional.  The reason is that we cache based on the `touch` value on `Classroom` and `ClassroomUnit`, and `ActivitySession` is configured to "bubble up" touch through those models.  Basically, what we're testing for here, is that it's bubbling up as intended.  It may make sense to add a comment here, though, so I'll do that.